include/fastdds/rtps/common/CacheChange.h

```diff
@@ -136,7 +136,7 @@ struct RTPS_DllAPI CacheChange_t
         serializedPayload.encapsulation = ch_ptr->serializedPayload.encapsulation;
 
         // Copy fragment size and initialize missing list to 'all fragments missing'
-        setFragmentSize(ch_ptr->fragment_size_, true);
+        setFragmentSize(ch_ptr->fragment_size_, false);
```

Comment:
May be a good idea to change the comment on the previous line to something like `// Just copy fragment size and calculate fragment count`

Reply:
OK